rustls/src/client/tls12.rs

```diff
@@ -25,12 +25,14 @@ use crate::client::common::{ClientAuthDetails, ReceivedTicketDetails};
 use crate::client::common::{HandshakeDetails, ServerCertDetails, ServerKXDetails};
 use crate::client::hs;
 
+use crate::internal::msgs::handshake::SessionID;
```

Comment:
This should be sorted with the other handshake type imports, and not go through the internal reexport.

Reply:
Oh, sure. I think Intellij did this automatically. Oops.